app/src/app/components/contents/explorer/Table.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import {PacketTable} from "../../../../types";
+
+export default function Table({data, headers}: PacketTable) {
```

Comment:
I'm a bit confused about why the headers are provided by the container component, `Explorer`, but then the table component makes assumptions about the contents of each column anyway, e.g. fetches `packet.displayName` for the first column. It feels like this should probably be providing its own headers.

Reply:
Header data is not moved to table component